internal/store/files/recog_place_store.go

```diff
@@ -75,6 +78,9 @@ type RecogPlaceStore struct {
 	DcidToNames map[string][]string
 	// Adjectival names with suffix.
 	AdjectivalNamesWithSuffix map[string]bool
+	// Map of reconName to list of possible types for reconNames that must be
```

Comment:
nit: As a reader its hard to know why we have a special reconName to list of types just from the name.  Perhaps rename it to `CommonWordReconNameToTypes` or something that gives a clue?

Reply:
We should probably also have comment in the code (like the stuff in the PR description) explaining this.

Reply:
done